src/applications/edu-benefits/sass/edu-benefits.scss

```diff
@@ -17,10 +17,9 @@
 
 // Privacy agreement?
 .form-required-span {
-  font-size: 1.7em;
```

Comment:
All the forms have the same privacy agreement checkbox, do we need to make this same change across the board?

Reply:
@jbalboni Let me double check this. When I was checking it yesterday it seemed like this one was the only one that was extra large. I think the previous css made the asterisk large because it was only the asterisk and the the Required text.